delta/sdk-testkit/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/testkit/ProjectsDummy.scala

```diff
@@ -130,6 +134,18 @@ final class ProjectsDummy private (
 
   override def currentEvents(offset: Offset): fs2.Stream[Task, Envelope[ProjectEvent]] =
     journal.currentEvents(offset)
+
+  override def provisionProject(subject: Subject): UIO[Unit] = subject match {
+    case user @ User(subject, realm) if provisioningConfig.enabled =>
+      (for {
+        org       <- IO.fromOption(provisioningConfig.enabledReams.get(realm))
+        proj      <- IO.fromEither(Label.apply(subject)).mapError { _ => () }
```

Comment:
Is this error mappings going to give us any information about the failure reason?

Reply:
I don't think we need that in the dummy implementation, but I can add it.

Reply:
Oh sorry, my bad. I didn't realize that's a dummy